tests/integration/targets/virt/templates/test_domain.xml.j2

```diff
@@ -0,0 +1,51 @@
+<domain type='qemu'>
+  <name>{{ domain_info['name'] }}</name>
+  <uuid>125388f2-6ab4-4343-a166-bad6ffad017e</uuid>
```

Comment:
I think we could prob drop `uuid` and it will get generated automatically when the VM is defined, this will at least keep it unique with each test run in case they are run in parallel somehow down the track

Reply:
This would cause the first test for "not changed" to fail - since the diff is really a syntactic diff (rather than semantic diff)